clang/include/clang/CIR/Dialect/IR/CIRTypes.h

```diff
@@ -184,6 +184,7 @@ class StructType
 
 bool isAnyFloatingPointType(mlir::Type t);
 bool isFPOrFPVectorTy(mlir::Type);
+bool isCIRIntOrIntVectorTy(mlir::Type);
```

Comment:
nit: to follow the pattern, should this be `isIntOrIntVectorTy `? It's already inside the CIR namespace anyways!

Reply:
makes sense, just changed name.